### PR TITLE
Add informations about the mail() method not being used anymore

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add informations about the mail() method not being used anymore

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Review by jolelievre — COMMENTED** [review]
Does it mean the shop should be configured to use SMTP?

**Comment by atomiix** [comment]
@jolelievre not necessarily, `/usr/sbin/sendmail` is a valid method too and it's what is used when calling `mail()` on some systems

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 1]
```diff
@@ -224,6 +224,13 @@ Modern modules that use Symfony controller and routing can now use the `$tabs` p
 
 Also new in this version, when a module is enabled/disabled all its related [Tab links are automatically shown/hidden]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-hiding-of-disabled-modules" >}}).
 
+## Default mail method does not use the `mail()` function anymore
+
+The `mail()` function is not used anymore [for security reasons](https://github.com/swiftmailer/swiftmailer/issues/866#issuecomment-289291228).
+Instead, `/usr/sbin/sendmail` took the default way of sending mails.
+{{% notice warning %}}
+You should check that you environment is configured to allow this use or change this settings in **Configure** > **Advanced Parameters** > **E-mail**.
```
Comment (jolelievre) [inline src/content/1.7/modules/core-updates/1.7.7.md:232]:
```suggestion
You should check that you environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```

Comment (jolelievre) [inline src/content/1.7/modules/core-updates/1.7.7.md:232]:
```suggestion
You should check that your server environment is configured to allow this use or change this settings in **Configure** > **Advanced Parameters** > **E-mail**.
```

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 2]
```diff
@@ -224,6 +224,13 @@ Modern modules that use Symfony controller and routing can now use the `$tabs` p
 
 Also new in this version, when a module is enabled/disabled all its related [Tab links are automatically shown/hidden]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-hiding-of-disabled-modules" >}}).
 
+## Default mail method does not use the `mail()` function anymore
+
+The `mail()` function is not used anymore [for security reasons](https://github.com/swiftmailer/swiftmailer/issues/866#issuecomment-289291228).
+Instead, `/usr/sbin/sendmail` took the default way of sending mails.
```
Comment (jolelievre) [inline src/content/1.7/modules/core-updates/1.7.7.md:230]:
```suggestion
Instead, `/usr/sbin/sendmail` is now the default way of sending mails.
```

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 3]
```diff
@@ -224,6 +224,13 @@ Modern modules that use Symfony controller and routing can now use the `$tabs` p
 
 Also new in this version, when a module is enabled/disabled all its related [Tab links are automatically shown/hidden]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-hiding-of-disabled-modules" >}}).
 
+## Default mail method does not use the `mail()` function anymore
+
+The `mail()` function is not used anymore [for security reasons](https://github.com/swiftmailer/swiftmailer/issues/866#issuecomment-289291228).
+Instead, `/usr/sbin/sendmail` is now the default way of sending mails.
+{{% notice warning %}}
+You should check that you environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```
Comment (jolelievre) [inline src/content/1.7/modules/core-updates/1.7.7.md:232]:
```suggestion
You should check that your server environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```

Comment (atomiix) [inline src/content/1.7/modules/core-updates/1.7.7.md:232]:
```suggestion
You should check that your server environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```

**Review by jolelievre — APPROVED** [review]

**Pull request merged**